src/dispatch/plugins/dispatch_core/plugin.py

```diff
@@ -162,6 +163,23 @@ def get(
         return recommendation.documents
 
 
+class DispatchContactPlugin(ContactPlugin):
+    title = "Dispatch Plugin - Contact plugin"
+    slug = "dispatch-contact"
+    description = "Uses dispatch itself to resolve incident participants."
+    version = dispatch_plugin.__version__
+
+    author = "Netflix"
+    author_url = "https://github.com/netflix/dispatch.git"
+
+    def get(self, email, db_session=None):
+        return getattr(
```

Comment:
Why not just access the values here directly like:
```
i =  individual_service.get_by_email(db_session=db_session, email=email),
return {"email": i.email, "fullname": i.email}
```

Reply:
It may return None in case the individual does not exist, e.g when it tries to create the individual as it doesn't exist, it retrieves data from contact plugin (which this case individual table)